proposals/params-span.md

```diff
@@ -51,8 +51,9 @@ Two overloads cannot differ by `params` modifier alone.
 
 `params` parameters are marked in metadata with a `System.ParamArrayAttribute`.
 
-A `params ReadOnlySpan<T>` is implicitly `scoped` unless explicitly annotated with `[UnscopedRef]`.
-Within the `params` method, the compiler will use escape analysis to report diagnostics if the `scoped` span is captured or returned.
+A `params ReadOnlySpan<T>` is implicitly `scoped`, and can be declared `scoped` explicitly.
+The parameter _cannot_ be annotated with `[UnscopedRef]`.
+Within the `params` method, the compiler will use escape analysis to report diagnostics if the implicitly `scoped` span is captured or returned.
```

Comment:
> if the implicitly `scoped` span is captured or returned

and not if the *explicitly* `scoped` span is captured?

Also why can it be explicitly marked `scoped`, isn't it confusing?